src/main/groovy/org/mockito/release/notes/improvements/GitHubTicketFetcher.java

```diff
@@ -215,8 +132,17 @@ public GitHubIssuesBuilder labels(String labels) {
                 return this;
             }
 
-            public GitHubIssues browse() {
-                return new GitHubIssues(authToken, state, filter, labels, direction);
+            GitHubIssues browse() {
+                // see API doc: https://developer.github.com/v3/issues/
+                String nextPageUrl = String.format("%s%s%s%s%s%s",
+                        "https://api.github.com/repos/mockito/mockito/issues?access_token=" + authToken,
```

Comment:
@TimvdLippe I think the best solution is when each attribute - value pair is a separate argument. It's easier to read.